test/integration/image-component/image-from-node-modules/test/index.test.js

```diff
@@ -0,0 +1,27 @@
+/* eslint-env jest */
+import { findPort, killApp, launchApp } from 'next-test-utils'
+import webdriver from 'next-webdriver'
+import { join } from 'path'
+
+const appDir = join(__dirname, '../')
+let appPort
+let app
+let browser
+
+// #31065
+describe('Image Component Tests In Dev Mode', () => {
```

Comment:
We also need a test for Prod Mode

Reply:
Done, I add this test case to prod env too